### PR TITLE
Add annotations for Arkouda and spectal norm

### PR DESCRIPTION
**Pull request by bmcdonald3**

### CODE REVIEW

**Comment by ronawho** [comment]
Dates are missing colons so annotation generation is failing. Missing chapcs OS reinstall?

**Comment by bmcdonald3** [comment]
Every time... thanks Elliot

File: test/ANNOTATIONS.yaml [hunk 1]
```diff
@@ -1835,6 +1838,8 @@ spectralnorm:
     - qthreads updated to yield every ~100 uncontested sync var locks
   12/18/18:
     - brad version changed to serial implementation (#11904)
+  02/22/22:
```
Comment (ronawho) [inline test/ANNOTATIONS.yaml:1841]: I'd go with the date we saw the impact for most configs. So probably the 18th -- https://chapel-lang.org/perf/chapcs/no-local/?startdate=2022/02/11&enddate=2022/02/24&configs=default,nolocal&graphs=spectralnormshootoutbenchmark

Comment (bmcdonald3) [reply test/ANNOTATIONS.yaml]: Alright, updated. (I was confused by what you were saying about this in the meeting, but understand now)

**Review by ronawho — APPROVED** [review]

**Pull request merged**